Samples/REST Services/Fill Address Form with Autocomplete/Fill Address Form with Autocomplete.html

```diff
@@ -1,163 +1,171 @@
+    <script src="https://atlas.microsoft.com/sdk/javascript/mapcontrol/3/atlas.min.js"></script>
+
+    <script>
+        //Your Azure Maps SAS token service URL for authenticating the service.
+        var tokenServiceUrl = 'https://samples.azuremaps.com/api/GetAzureMapsSasToken';
+
+        var token;
+
+        var addressGeocodeServiceUrlTemplate = 'https://atlas.microsoft.com/geocode:autocomplete?api-version=2025-06-01-preview&query={query}&coordinates={coordinates}&language={language}&resultTypeGroups={resultTypeGroups}&resultTypes={resultTypes}&countryRegion={countryRegion}&top={top}';
+
+        async function PageLoaded() {
+            //Retrieve a SAS token if there isn't one currently available.
+            if (!token) {
+                await initSasToken();
+            }
+
+            //Create a jQuery autocomplete UI widget.
+            $("#searchBox").autocomplete({
+                minLength: 3,   //Don't ask for suggestions until at least 3 characters have been typed. This will reduce costs by not making requests that will likely not have much relevance.
+                source: function (request, response) {
+
+                    //Create a URL to the Azure Maps search service to perform the address search.
+                    var requestUrl = addressGeocodeServiceUrlTemplate.replace('{query}', encodeURIComponent(request.term))
+                        .replace('{coordinates}', '-95.712891,37.09024') // Default set to US center coordinates
+                        .replace('{language}', 'en-US')
+                        .replace('{resultTypeGroups}', 'Address')
+                        .replace('{resultTypes}', 'Address')
+                        .replace('{countryRegion}', 'US')
+                        .replace('{top}', '10'); // Limit to top 10 results
+
+                    //Process the request.
+                    fetch(requestUrl, {
+                        method: 'GET',
+                        mode: 'cors',
+                        headers: {
+                            'Authorization': 'jwt-sas ' + token,
+                            'Content-Type': 'application/json; charset=utf-8',
+                            'Accept-Language': 'en-US'
+                        }
+                    }).then(r => r.json()).then(data => {
+                        // API already filters for Address type items with resultTypeGroups and resultTypes parameters
+                        var results = data.features || [];
+                        response(results);
+                    });
+                },
+                select: function (event, ui) {
+                    //When a suggestion has been selected.
+                    var selection = ui.item;
+                    var address = selection.properties.address;
+
+                    //Populate the address textbox values - mapping new API structure to old fields
+                    document.getElementById('addressLineTbx').value = address.addressLine || 

```

Comment:
`address.addressLine` seems to be missing the street name and only includes the street number?

<img width="858" height="308" alt="Screenshot 2025-09-24 152239" src="https://github.com/user-attachments/assets/63fe8c26-6054-4761-8429-1501a8015193" />

<img width="628" height="321" alt="Screenshot 2025-09-24 152257" src="https://github.com/user-attachments/assets/d8ad886a-f671-41c6-97b9-ed31c9d4291a" />